templates/Calculate_Mixing_Scores.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    neighbors_file_name = 'neighborhood_counts-cell_meta_cluster_50.csv'
</pre><p>I think we should add "radius" before the integer amount to make the file name more clear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>